src/controllers/rsvpController.js

```diff
@@ -1,56 +1,65 @@
 import rsvp from '../models/rsvp';
-import rsvps from '../data/rsvp.json';
 import meetup from '../models/Meetup';
-import helper from '../helpers/helper';
 
 class RsvpController {
   static index(request, response){
-    const returnedRsvps = rsvp.all(request.params.id);
-    if(returnedRsvps.length > 0) {
+    rsvp.all(request.params.id)
+    .then(response => {
       return response.status(200).json({
         status: 200,
-        data: returnedRsvps
-      })
-    }
-    return response.status(404).json({
-      status: 404,
-      error: `No RSVP exists for this meetup`
+        data: response.body
+      })      
+    })
+    .catch(error => {
+      return response.status(404).json({
+        status: 404,
+        error: error.message
+      })    
     })
   }
 
   static create(request, response) {
-    if (meetup.all().length > 0) {
-      if (request.body.user && request.body.status) {
-        const rsvpMeetup = meetup.find(request.params.id);
-        if (rsvpMeetup instanceof Object) {
-          const newRsvp = {
-            id: helper.getNewId(rsvps),
-            meetup: parseInt(rsvpMeetup.id),
-            topic: rsvpMeetup.title,
-            status: request.body.status,
-            user: parseInt(request.body.user),
-          };
-          const createdRsvp = rsvp.create(newRsvp);
-          return response.status(201).json({
-            status: 201,
-            data: createdRsvp,
-          });
-        }
-        return response.status(404).json({
+    const userId = request.user.id;
+    const payload = {
+      userId,
+      meetupId: request.params.id,
+      topic: request.body.topic,
+      status: request.body.status
+    }
+    rsvp.create(payload)
+    .then(response => {
+      return response.status(201).json({
+        status: 201,
+        data: response.body
+      })      
+    })
+    .catch(error => {
+      switch (error.status) {
```

Comment:
I think this section can be an helper function, just so you can use it in more than one place to avoid code smells